docs/kubectl.md

```diff
@@ -390,6 +390,7 @@ Additional help topics:
   kubectl run-container Run a particular image on the cluster.
   kubectl stop          Gracefully shutdown a resource
   kubectl expose        Take a replicated application and expose it as Kubernetes Service
+  kubectl label         Update the labels on a resource
```

Comment:
At some point we should be worried about verb overload for new users - I think one lesson we took from our previous clients was that as the number of objects increase, having multiple verb forms of a command started to crowd the limited space in cli help.  When dealing with concepts that were common across all objects, having add-X and remove-X commands vs just the single "work with X" command detracted from initial usability.


> On Feb 7, 2015, at 11:48 AM, Brian Grant <notifications@github.com> wrote:
> 
> In docs/kubectl.md:
> 
> > @@ -390,6 +390,7 @@ Additional help topics:
> >    kubectl run-container Run a particular image on the cluster.
> >    kubectl stop          Gracefully shutdown a resource
> >    kubectl expose        Take a replicated application and expose it as Kubernetes Service
> > +  kubectl label         Update the labels on a resource
> Did you decide not to add commands to remove or change labels? Or do you imagine that would be done via flags?
> 
> —
> Reply to this email directly or view it on GitHub.
>